src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
@patflynn, @loosebazooka - I updated this:

- For devappserver1, the client supplied env vars are set on the running process
- For devappserver2, the client supplied env vars are passed into the --env_var argument in the format of my previous comment

please take a look